packages/security_center/lib/services/disk_encryption_service.dart

```diff
@@ -4,6 +4,19 @@ import 'package:security_center/disk_encryption/disk_encryption_providers.dart';
 part 'disk_encryption_service.freezed.dart';
 part 'disk_encryption_service.g.dart';
 
+/// A Class to model an Entropy response from Snapd
+@freezed
+class EntropyResponse with _$EntropyResponse {
+  const factory EntropyResponse({
+    required int entropyBits,
+    required int minEntropyBits,
+    required int optimalEntropyBits,
+  }) = _EntropyResponse;
```

Comment:
I'd propose using a richer `EntropyResponse` structure, similar to what we're working on in the installer, see https://github.com/canonical/ubuntu-desktop-provision/pull/1085/files#diff-25c90a4ccb338320620a8697e0e979eaa2bed04dab5cc8e1ae4f5774089fc568 from https://github.com/canonical/ubuntu-desktop-provision/pull/1085.

Once the improved API lands, snapd will respond with an error if the check fails, which we should use as a direct indicator instead of performing comparisons on the provided entropy data in the UI.